script.js

```diff
@@ -1,3 +1,16 @@
+var rollDice = function () {
+  return 1;
```

Comment:
Just a minor thing, but perhaps you could remove this result manipulation statements for further pushes of code just in case haha